modules/ROOT/pages/custom-ingress-configuration.adoc

```diff
@@ -0,0 +1,52 @@
+= Defining a Custom Ingress Configuration
+
+[NOTE]
+====
+The tasks outline in this topic must be performed by an IT administrator
+====
+
+Runtime Fabric on Self-managed Kubernetes enables you to edit a template file that specifies a custom Ingress configuration. Using this file you can define annotations, rewrite rules, the ingressClassName, and more.
+
+The template file is a Kubernetes ConfigMap file. After customizing the template, you can use the `rtfctl apply` command-line utility to apply the custom Ingress configuration to your Kubernetes cluster.
+
+If you do not apply a custom template file, Runtime Fabric uses a default configuration that works with the out-of-the-box Kubernetes Ingress configuration.
+
+== Procedure
+
+. Copy the following into a text file:
++
+----
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: ingress-config
+  namespace: rtf
+data:
+  ingress.annotations: |
+    nginx.ingress.kubernetes.io/rewrite-target: /
+    nginx.ingress.kubernetes.io/ssl-redirect: true
+    kubernetes.io/ingress.class: <name>
+  ingress.spec: |
+    ingressClassName: <name>
+    rules.http.paths.path: /${APP_PATH}
+----
+
+. Edit the text file as required.
++
+All changes should be made under the `data` section of this file. Do not modify any other sections of the template. The most common configuration options are.
++
```

Comment:
Can we tabulate this in a manner consistent with what we do elsewhere, for e.g.: https://docs.mulesoft.com/runtime-fabric/1.7/runtime-fabric-logs#azure-log-analytics-configuration-parameters

cc @chingyi-lin

Reply:
Yeah with that we can clearly distinguish between Required and Optional vars

Reply:
The problem is how do you represent the hierarchical relationship between elements? In the example you provided they are all at the same level.

Reply:
@sleslie23 They'll be at same level, so these can be defined in tabulated form.

Reply:
They are not at the same level in the file. This is the file I got from the spec. Please provide a revised version that contains exactly what we want the user modify.

Reply:
Reformatted as a table.